docs/SPEC.md

```diff
@@ -468,6 +468,12 @@ Examples:
     2018-07-01T00:00:00Z + 2y  // 2020-07-01T00:00:00Z
     2018-07-01T00:00:00Z + 5h  // 2018-07-01T05:00:00Z
 
+#### Binary types
+
+A _binary type_ represents a base64 encoded sequence of bytes as defined in RFC 4648.
+The binary type name is `bytes`.
+The binary type is nullable.
```

Comment:
This binary doesn't need to be base64 encoded. That was just how the csv format would represent the bytes type. Can you change this so say that it is a sequence of byte values? 

Also can you add it to the list of named types?